Automattic/wp-calypso
# Converting current theme bar display to new, more clear design

#### Changes proposed in this Pull Request

* This PR updates the "Current Theme" bar to the new design as described in https://github.com/Automattic/wp-calypso/issues/54075.

![image](https://user-images.githubusercontent.com/13437011/124662789-dd917b80-de6e-11eb-84fb-a06335610eca.png)


#### Testing instructions



* Navigate to `/themes/{SITE_URL}?flags=theme/showcase-revamp`
* Verify that the current theme banner displays as indicated above.
* Verify that previous (production) design does not display if feature flag is not enabled.



Related to #54075
